### PR TITLE
EDU-1149 Redesign homepage under the fold

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1149

Pictogram is removed, so that technically we can go live and are not blocked. For a short period, we probably can do without the pictogram, even though it makes a big difference.

Regarding the translations for the `homepage-project-presentation` main text and `homepage-oer-presentation` main text, the OMA versions are in a comment on the JIRA ticket. Please take a look over them too and adjust/correct them or let me know what to correct 🙏

### CODE REVIEW

File: src/components/homepage/homepage-oer-presentation.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  descriptionMarkdown:
+    en: |
+      Our goal is to create a community project that is carried and supported by as many people around the world as possible.
+      If you are inspired by the idea of free learning and of an open learning platform, then join our community for **free**.
+    de: |
+      Unser Ziel ist, ein Gemeinschaftsprojekt zu schaffen, das von möglichst vielen Menschen auf der ganzen Welt getragen und unterstützt wird.
+      Wenn du von der Idee des kostenlosen lernens und einer offenen Lernplattform begeistert bist uns selbst Inhalte generieren möchtest, trit unserer Community **kostenlos** bei.
+  signUp:
+    en: Sign up for your free account
+    de: Melde dich für dein kostenloses Konto an
```
Comment (adrianaluchian) [inline src/components/homepage/homepage-oer-presentation.yml:14]: "Registrier" statt "Melde" hier? Don't know if "Registrier dich" works though..

File: src/components/homepage/homepage-document-cards.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+homepageDocumentCards:
+  headline:
+    en: Popular documents
+    de: Popular documents
```
Comment (ahelmberger) [inline src/components/homepage/homepage-document-cards.yml:4]: Beliebte Dokumente

File: src/components/homepage/homepage-document-cards.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  contributorsTooltipPrefix:
+    en: Contributions by
+    de: Contributions by
```
Comment (ahelmberger) [inline src/components/homepage/homepage-document-cards.yml:7]: Beiträge von

File: src/components/homepage/homepage-oer-presentation.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  headline:
+    en: Become part of the OER community!
+    de: Werde Teil der OER-Community!
+  descriptionMarkdown:
+    en: |
+      Our goal is to create a community project that is carried and supported by as many people around the world as possible.
+      If you are inspired by the idea of free learning and of an open learning platform, then join our community for **free**.
+    de: |
+      Unser Ziel ist, ein Gemeinschaftsprojekt zu schaffen, das von möglichst vielen Menschen auf der ganzen Welt getragen und unterstützt wird.
+      Wenn du von der Idee des kostenlosen lernens und einer offenen Lernplattform begeistert bist uns selbst Inhalte generieren möchtest, trit unserer Community **kostenlos** bei.
```
Comment (ahelmberger) [inline src/components/homepage/homepage-oer-presentation.yml:11]: Wenn du von der Idee des kostenlosen **L**ernens und einer offenen Lernplattform begeistert bist un**d** selbst Inhalte generieren möchtest, trit**t** unserer Community kostenlos bei

_(corrections in bold)_

File: src/components/homepage/homepage-oer-presentation.yml (hunk 1 above)
Comment (ahelmberger) [reply src/components/homepage/homepage-oer-presentation.yml:14]: Yep, would work, e.g. "Registriere dich für dein kostenloses Konto"

File: src/components/homepage/homepage-project-presentation.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+homepageProjectPresentation:
+  descriptionMarkdown:
+    en: |
+      **Educandu** is the open platform for learning and teaching.
+      And everyone can contribute and make their knowledge availabe as open educational resources (OER) under a Creative Commons license.
+      Simple, intuitive and free, with features developed specifically for teaching:
+    de: |
+      **Educandu**, ist die Platform für digitales Lernen und Unerrichten.
```
Comment (ahelmberger) [inline src/components/homepage/homepage-project-presentation.yml:8]: The comma has to go here

File: src/components/homepage/homepage-project-presentation.yml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+      Und jeder kann mitmachen und kann seine Wissen als Open Educational Resource (OER) unter einer Creative Commons Lizenz zur Verfügung stellen.
```
Comment (ahelmberger) [inline src/components/homepage/homepage-project-presentation.yml:9]: "... sein Wissen ..."

File: src/components/homepage/homepage-project-presentation.yml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+      Einfach, intuitiv und kostenlos mit speziell für unerrichten entwickelten Features:
```
Comment (ahelmberger) [inline src/components/homepage/homepage-project-presentation.yml:10]: "... kostenlos, mit speziell für das Unterrichten entwickelten ..."

File: src/components/homepage/homepage-project-presentation.yml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  mediaFeature:
+    en: Video and audio with interactive options
+    de: Video und Audio mit interaktiven Möglichkeiten
+  earTrainingFeature:
+    en: Online ear training tasks
+    de: Online Gehörbildungsaufgaben
```
Comment (ahelmberger) [inline src/components/homepage/homepage-project-presentation.yml:16]: Connecting "Online" and "Gehörbildungsaufgaben" with a hyphen would be better.

File: src/components/homepage/homepage-project-presentation.yml [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    de: Mehrspur-Musikplayer
+  pdfFeature:
+    en: Embedded PDF viewer
+    de: Eingebauter PDF-Viewer
+  analysisFeature:
+    en: Music and movie analysis tool
+    de: Musik- und Filmanalysetool
+  textFeature:
+    en: Easy text editing and formatting
+    de: Einfache Texteingabe und- formatierung
```
Comment (ahelmberger) [inline src/components/homepage/homepage-project-presentation.yml:28]: The hyphen and the space have to switch their position here.

**Comment by ahelmberger** [comment]
> Closes https://educandu.atlassian.net/browse/EDU-1149
> 
> Pictogram is removed, so that technically we can go live and are not blocked. For a short period, we probably can do without the pictogram, even though it makes a big difference.
> 
> Regarding the translations for the `homepage-project-presentation` main text and `homepage-oer-presentation` main text, the OMA versions are in a comment on the JIRA ticket. Please take a look over them too and adjust/correct them or let me know what to correct pray

Looks very nice (imagining the pictogram for now). There's one thing we'll want to look at (and you have probably missed that on the Mac): There's a permanently visible horizontal scrollbar to compensate the ~17px the new vertical scrollbar takes away from the `100vw`. I've tried in the dev tools checking if I could "easily" pinpoint it to some style rule, but I could just make it smaller, not disappear. I don't think it's necessary to correct it in this PR though, as we have to revisit the page once more again anyways, it's up to you.

File: src/components/homepage/homepage-oer-presentation.yml (hunk 4 above)
Comment (adrianaluchian) [reply src/components/homepage/homepage-oer-presentation.yml]: Thank you! I have corrected everything you found 🙏

Comment (ahelmberger) [reply src/components/homepage/homepage-oer-presentation.yml]: :+1:

**Comment by adrianaluchian** [comment]
> 
> Looks very nice (imagining the pictogram for now). There's one thing we'll want to look at (and you have probably missed that on the Mac): There's a permanently visible horizontal scrollbar to compensate the ~17px the new vertical scrollbar takes away from the `100vw`. I've tried in the dev tools checking if I could "easily" pinpoint it to some style rule, but I could just make it smaller, not disappear. I don't think it's necessary to correct it in this PR though, as we have to revisit the page once more again anyways, it's up to you.

If you want to get an idea how it is with the pictogram, you can check out commit `ab065316b816d4d37a59b00aa5cbd3dc7addd1c1` which is before the pictogram removal (and before a few more small changes).

Interesting about the scrollbar 🤔 I definitely don't see it on my end, not without actually trying to scroll
I will check if it's something easy to pinpoint.

**Comment by ahelmberger** [comment]
> > Looks very nice (imagining the pictogram for now). There's one thing we'll want to look at (and you have probably missed that on the Mac): There's a permanently visible horizontal scrollbar to compensate the ~17px the new vertical scrollbar takes away from the `100vw`. I've tried in the dev tools checking if I could "easily" pinpoint it to some style rule, but I could just make it smaller, not disappear. I don't think it's necessary to correct it in this PR though, as we have to revisit the page once more again anyways, it's up to you.
> 
> If you want to get an idea how it is with the pictogram, you can check out commit `ab065316b816d4d37a59b00aa5cbd3dc7addd1c1` which is before the pictogram removal (and before a few more small changes).
> 
> Interesting about the scrollbar thinking I definitely don't see it on my end, not without actually trying to scroll I will check if it's something easy to pinpoint.
> 
> **Update:** If you were talking about mobile, the culprit is the much too long "Registriere dich für dein kostenloses Konto" button text sweat_smile Any way we could rephrase it shorter in the same manner? Maybe towards "Kostenloses Konto Registrierung" (but in correct German)?

The scrollbars are always visible (on Win+Linux, I guess) regardless of the screen width, unfortunately. Concerning the button text, we could shorten it to "Jetzt kostenlos registrieren".

**Comment by adrianaluchian** [comment]
> > > Looks very nice (imagining the pictogram for now). There's one thing we'll want to look at (and you have probably missed that on the Mac): There's a permanently visible horizontal scrollbar to compensate the ~17px the new vertical scrollbar takes away from the `100vw`. I've tried in the dev tools checking if I could "easily" pinpoint it to some style rule, but I could just make it smaller, not disappear. I don't think it's necessary to correct it in this PR though, as we have to revisit the page once more again anyways, it's up to you.
> > 
> > 
> > If you want to get an idea how it is with the pictogram, you can check out commit `ab065316b816d4d37a59b00aa5cbd3dc7addd1c1` which is before the pictogram removal (and before a few more small changes).
> > Interesting about the scrollbar thinking I definitely don't see it on my end, not without actually trying to scroll I will check if it's something easy to pinpoint.
> > **Update:** If you were talking about mobile, the culprit is the much too long "Registriere dich für dein kostenloses Konto" button text sweat_smile Any way we could rephrase it shorter in the same manner? Maybe towards "Kostenloses Konto Registrierung" (but in correct German)?
> 
> The scrollbars are always visible (on Win+Linux, I guess) regardless of the screen width, unfortunately. Concerning the button text, we could shorten it to "Jetzt kostenlos registrieren".

Very good find, thanks for this! 🙏 
I've fixed it along with another overflow case on mobile.

**Comment by ahelmberger** [comment]
> > > > Looks very nice (imagining the pictogram for now). There's one thing we'll want to look at (and you have probably missed that on the Mac): There's a permanently visible horizontal scrollbar to compensate the ~17px the new vertical scrollbar takes away from the `100vw`. I've tried in the dev tools checking if I could "easily" pinpoint it to some style rule, but I could just make it smaller, not disappear. I don't think it's necessary to correct it in this PR though, as we have to revisit the page once more again anyways, it's up to you.
> > > 
> > > 
> > > If you want to get an idea how it is with the pictogram, you can check out commit `ab065316b816d4d37a59b00aa5cbd3dc7addd1c1` which is before the pictogram removal (and before a few more small changes).
> > > Interesting about the scrollbar thinking I definitely don't see it on my end, not without actually trying to scroll I will check if it's something easy to pinpoint.
> > > **Update:** If you were talking about mobile, the culprit is the much too long "Registriere dich für dein kostenloses Konto" button text sweat_smile Any way we could rephrase it shorter in the same manner? Maybe towards "Kostenloses Konto Registrierung" (but in correct German)?
> > 
> > 
> > The scrollbars are always visible (on Win+Linux, I guess) regardless of the screen width, unfortunately. Concerning the button text, we could shorten it to "Jetzt kostenlos registrieren".
> 
> Very good find, thanks for this! pray I've fixed it along with another overflow case on mobile.

Oh nice, works perfectly now (on all screen widths) :heart_eyes: Great job!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**